poco/lua/ClientConnection.lua

```diff
@@ -0,0 +1,115 @@
+
```

Comment:
We're using the Cocos2Dx-lua api.
We have a modified lua module though (defold-poco.lua)